javaslang/src/test/java/javaslang/collection/TraverableTest.java

```diff
@@ -0,0 +1,45 @@
+package javaslang.collection;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+
+public class TraverableTest {
+
+    // -- max()
+
+    @Test
+    public void maxShouldBeEmpty() {
+         assertTrue(Iterator.empty().max().isEmpty());
+    }
+
+    @Test
+    public void maxShouldBeTheOnlyValue() {
+        assertEquals(Integer.valueOf(5), Iterator.of(5).max().get());
```

Comment:
To align it with the other tests we would write:

```java
assertThat(of(5).max()).isEqualTo(Option.some(5));
```

The benefit is that we additionally verify that a Some(5) is returned. Otherwise (technically) the test would be also green if we change the return type to Try and we return a Success(5).